src/astroprint/api/printer.py

```diff
@@ -497,3 +497,36 @@ def sendComm():
 
 	else:
 		return make_response("Command is missing", 400)
+
+
+@api.route("/printer/allowed-features", methods=["GET"])
+def getAllowedFeatures():
+	return jsonify(printerManager().getAllowedFeatures())
+
+@api.route("/printer/load-filament", methods=["POST"])
```

Comment:
It would be better that the api endpoint be some more generic. Like a /printer/execute-routine